apps-devstg/us-east-1/dynamodb-access-rules-demoapps --/demo_client/main.py

```
@@ -0,0 +1,464 @@
+    """
+    try:
+        # Instantiate the DynamoDB client using the temporary credentials
+        dynamodb = boto3.resource('dynamodb',
+            region_name=REGION,
+            aws_access_key_id=credentials['AccessKeyId'],
+            aws_secret_access_key=credentials['SecretKey'],
+            aws_session_token=credentials['SessionToken']
+        )
+
+        table = dynamodb.Table(DYNAMODB_TABLE_NAME)
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)